mitodl/ocw-hugo-themes
# fix instructor insights styles

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/63

#### What's this PR do?
In the transition to `ocw-hugo-themes`, the `instructor_insights` styles were broken.  This PR includes them again alongside the course theme in `extrahead.html` if the current url contains `instructor-insights`.

#### How should this be manually tested?
 - Read the readme for this project if you have not spun up a course locally before
 - Spin up a course site for `6-034-artificial-intelligence-fall-2010`
 - Verify that `instructor_insights.css` is being loaded in your inspector, and ensure that the pages look like the original designs:
   - https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/441780346/preview?scrollOffset=999
   - https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/442724922/preview?scrollOffset=999

#### Screenshots (if appropriate)
![chrome_2021-05-21_15-05-40](https://user-images.githubusercontent.com/12089658/119186407-10f68300-ba46-11eb-9050-f60a6fa6939f.png)
![image](https://user-images.githubusercontent.com/12089658/119186614-561ab500-ba46-11eb-8583-a0beacb18b5b.png)
![image](https://user-images.githubusercontent.com/12089658/119186669-6af74880-ba46-11eb-991f-92e0f9c3bfc0.png)
![image](https://user-images.githubusercontent.com/12089658/119186803-82363600-ba46-11eb-8f37-1a6c27d6c436.png)
